src/hooks/brick_attached.rs

```diff
@@ -4,10 +4,13 @@ use std::path::PathBuf;
 use super::super::initialize_storage;
 
 pub fn brick_attached() -> Result<(), String> {
-    let brick_location = juju::storage_get_location().map_err(|e| e.to_string())?;
-    let brick_path = PathBuf::from(brick_location.trim());
+    //let brick_location = juju::storage_get_location().map_err(|e| e.to_string())?;
+    //let brick_path = PathBuf::from(brick_location.trim());
 
     // Format our bricks and mount them
-    initialize_storage(&brick_path)?;
-    Ok(())
+    //initialize_storage(&brick_path)?;
+    //Ok(())
```

Comment:
Yeah I decided to see what would happen if I skipped the brick_attached hook and did the juju storage drives in parallel.  It worked fine so I'm just going to remove that hook.